.nais/naiserator-prod-gcp.yaml

```diff
@@ -30,10 +30,10 @@ spec:
     path: /internal/prometheus
   resources:
     limits:
-      memory: 1024Mi
+      memory: 2048Mi
     requests:
       cpu: 50m
-      memory: 128Mi
+      memory: 2048Mi
```

Comment:
Er denne en litt hissig økning? 😅 Vet ikke helt hva konsekvensen er da, men virker litt mye å øke fra `128` til `2048`

Reply:
Ja, vi har kanskje tatt for hardt i her, men kan redusere når vi ser om det løser problemet med meldingen som feiler nå.